husarion_ugv_controller/launch/controller.launch.py

```diff
@@ -277,6 +289,11 @@ def generate_launch_description():
             "controller_manager",
             "--controller-manager-timeout",
             "10",
+            "--ros-args",
```

Comment:
Is it actually gives something?
This is only `spawner` so I think we can skip debug logs here.

Reply:
Removed